docs/common/navigation-data.js

```diff
@@ -123,6 +72,15 @@ const generateNavLinks = (path_, arr) => {
   return arr;
 };
 
-module.exports = path_ => {
-  return sortNav(generateNavLinks(path_, []));
-};
+const ORIGINAL_PATH_PREFIX = './versions';
+
+let versionDirectories = fs
+  .readdirSync(ORIGINAL_PATH_PREFIX, { withFileTypes: true })
+  .filter(f => f.isDirectory())
+  .map(f => f.name);
+
+module.exports = versionDirectories.reduce(
+  (obj, version) =>
+    Object.assign({ [version]: generateNavLinks(`${ORIGINAL_PATH_PREFIX}/${version}`, []) }, obj),
```

Comment:
`{ ...obj, [version]: generateNavEtc }` seems a bit more clean to me but don't mind so much either way

Reply:
Oh, good idea.